examples/common.yaml

```diff
@@ -32,6 +32,7 @@ openstack::mysql::allowed_hosts: ['localhost', '127.0.0.1', '172.16.33.%']
 
 openstack::rabbitmq::user: 'openstack'
 openstack::rabbitmq::password: 'pose-vix'
+openstack::rabbitmq::hosts: ['localhost:5672', '127.0.0.1:5672']
```

Comment:
This should be ['172.16.33.4:5672'] instead of localhost, otherwise the compute node will try to connect to the rabbitmq server on localhost in the example multinode case. Could you also add this to examples/allinone.yaml so that the allinone example still works?